src/components/pages/pages.test.tsx

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable react/prop-types */
```

Comment:
suggestion: remove the eslint disable here

I don't believe it's required for this file and therefore can just be removed